test/regression/api/canary/admin/settings_spec.js

```diff
@@ -678,7 +678,7 @@ describe('Settings API (canary)', function () {
             jsonResponse.settings.length.should.eql(1);
             testUtils.API.checkResponseValue(jsonResponse.settings[0], ['id', 'group', 'key', 'value', 'type', 'flags', 'created_at', 'updated_at']);
             jsonResponse.settings[0].key.should.eql('labs');
-            jsonResponse.settings[0].value.should.eql(JSON.stringify({}));
+            jsonResponse.settings[0].value.should.match(/^\{.*\}$/);
```

Comment:
Why was this change needed?

Reply:
The value was populated with all of the labs settings so we couldn't do an `===` on `'{}'`. As the labs settings will change over time I figured it would be less of a headache to expect an object-like JSON string rather than a direct match.

Reply:
Makes sense :+1:  Let's extract this regex into a vairable. Find it hard looking at event the simplest regexes without breaking out of the context :sweat_smile:

Reply:
👍 done